### ISSUE TITLE
Push new version to container registry

### ISSUE THREAD

**Issue by samuelrade** [8gears/n8n-helm-chart#124]
Probably @Vad1mo; could you push the latest stable version to ArtfactHUB [8gears.container-registry.com](http://8gears.container-registry.com/) by triggering [this](https://github.com/8gears/n8n-helm-chart/actions/workflows/push.yaml) workflow?

**Comment by mayphilc** [comment]
im also in need of the update please push the latest stable release

**Comment by Vad1mo** [comment]
why is it important for you to have a new chart version for each n8n release? 
You can just go ahead and set the `appVersion` to the version you need, this is what I do all the time.

**Comment by Jorgagu** [comment]
👋 @Vad1mo, you're absolutely right ! It is possible to use the `appVersion` for this. However, it's generally considered good practice to align chart releases with app versions, as certain app versions might require new adjustments or configurations.

That said, if aligning versions perfectly isn't feasible, it's still a good idea to create a release—even a minor one—when there are no breaking changes.

Additionally, we could consider adding [Renovate Bot](https://github.com/marketplace/renovate) to automate the process of updating dependencies. What do you think?

**Comment by Vad1mo** [comment]
> Additionally, we could consider adding [Renovate Bot](https://github.com/marketplace/renovate) to automate the process of updating dependencies. What do you think?

Yeah that would be an option too, if you know the tool, you can propose a PR, I'll happily accept that

**Comment by Jorgagu** [comment]
Hi @Vad1mo,
Since I'm not an owner or member of 8gears, I don't have the necessary permissions to add the Renovate bot to the repository.
Could you please initialize the Renovate bot in the repo or organization?
Once it's set up, I can take care of the configuration and submit a PR if you'd like.
